src/libraries/System.Security.Cryptography.Algorithms/src/Internal/Cryptography/RandomNumberGeneratorImplementation.Browser.cs

```diff
@@ -0,0 +1,31 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.IO;
+using System.Diagnostics;
+
+namespace System.Security.Cryptography
+{
+    internal partial class RandomNumberGeneratorImplementation
+    {
+        private static readonly FileStream s_randomStream = new FileStream("/dev/random", FileMode.Open, FileAccess.Read);
```

Comment:
Should this use urandom?  Wondering if we might ever hit cases where there's not enough entropy available and reading ends up blocking.

Reply:
On emscripten, both are redirected to the same random source.